### ISSUE TITLE
`java.lang.UnsupportedOperationException` Error using Infernal Mobs

### ISSUE THREAD

**Issue by zisn** [AtomicStryker/atomicstrykers-minecraft-mods#518]
Hello! I just started a server to play with some friends. The mod is only added as `server-side`, but I'm getting constant crashes. This one was the last. I cannot find a way to fix it.

I don't know if somebody can bring me a hand figuring out what's going on! Thank you in advance :D

I'm using:
- NeoForge: `21.1.45`
- JDK: `21.0.4`
- Minecraft: `1.21.1`
- Infernal Mobs version: `infernalmobs-1.21.3NF`
- RAM: `32GB`

```log
---- Minecraft Crash Report ----
// Daisy, daisy...

Time: 2024-09-09 16:43:17
Description: Ticking entity

java.lang.UnsupportedOperationException: Payload infernalmobs:velocity may not be sent to the client!
	at TRANSFORMER/neoforge@21.1.45/net.neoforged.neoforge.network.registration.NetworkRegistry.checkPacket(NetworkRegistry.java:445) ~[neoforge-21.1.45-universal.jar%23137!/:?] {re:classloading}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.network.ServerCommonPacketListenerImpl.send(ServerCommonPacketListenerImpl.java:178) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,re:mixin}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.network.ServerCommonPacketListenerImpl.send(ServerCommonPacketListenerImpl.java:173) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,re:mixin}
	at TRANSFORMER/neoforge@21.1.45/net.neoforged.neoforge.network.PacketDistributor.sendToPlayer(PacketDistributor.java:51) ~[neoforge-21.1.45-universal.jar%23137!/:?] {re:classloading}
	at TRANSFORMER/infernalmobs@1.21.3NF/atomicstryker.infernalmobs.common.InfernalMobsCore.sendVelocityPacket(InfernalMobsCore.java:776) ~[infernalmobs-1.21.3NF.jar%23144!/:1.21.3NF] {re:classloading}
	at TRANSFORMER/infernalmobs@1.21.3NF/atomicstryker.infernalmobs.common.mods.MM_Blastoff.tryAbility(MM_Blastoff.java:66) ~[infernalmobs-1.21.3NF.jar%23144!/:1.21.3NF] {re:classloading}
	at TRANSFORMER/infernalmobs@1.21.3NF/atomicstryker.infernalmobs.common.mods.MM_Blastoff.onUpdate(MM_Blastoff.java:37) ~[infernalmobs-1.21.3NF.jar%23144!/:1.21.3NF] {re:classloading}
	at TRANSFORMER/infernalmobs@1.21.3NF/atomicstryker.infernalmobs.common.MobModifier.onUpdate(MobModifier.java:249) ~[infernalmobs-1.21.3NF.jar%23144!/:1.21.3NF] {re:classloading}
	at TRANSFORMER/infernalmobs@1.21.3NF/atomicstryker.infernalmobs.common.EntityEventHandler.onEntityLivingUpdate(EntityEventHandler.java:175) ~[infernalmobs-1.21.3NF.jar%23144!/:1.21.3NF] {re:classloading}
	at MC-BOOTSTRAP/net.neoforged.bus/net.neoforged.bus.EventBus.post(EventBus.java:350) ~[bus-8.0.1.jar%2367!/:?] {}
	at MC-BOOTSTRAP/net.neoforged.bus/net.neoforged.bus.EventBus.post(EventBus.java:315) ~[bus-8.0.1.jar%2367!/:?] {}
	at TRANSFORMER/neoforge@21.1.45/net.neoforged.neoforge.event.EventHooks.fireEntityTickPost(EventHooks.java:944) ~[neoforge-21.1.45-universal.jar%23137!/:?] {re:classloading}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.level.ServerLevel.tickNonPassenger(ServerLevel.java:775) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.world.level.Level.guardEntityTick(Level.java:570) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.level.ServerLevel.lambda$tick$2(ServerLevel.java:420) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.world.level.entity.EntityTickList.forEach(EntityTickList.java:54) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.level.ServerLevel.tick(ServerLevel.java:400) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1037) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:317) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:917) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:707) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.lambda$spin$2(MinecraftServer.java:267) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at java.base/java.lang.Thread.run(Thread.java:1583) [?:?] {re:mixin}


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server thread
Stacktrace:
	at TRANSFORMER/neoforge@21.1.45/net.neoforged.neoforge.network.registration.NetworkRegistry.checkPacket(NetworkRegistry.java:445) ~[neoforge-21.1.45-universal.jar%23137!/:?] {re:classloading}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.network.ServerCommonPacketListenerImpl.send(ServerCommonPacketListenerImpl.java:178) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,re:mixin}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.network.ServerCommonPacketListenerImpl.send(ServerCommonPacketListenerImpl.java:173) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,re:mixin}
	at TRANSFORMER/neoforge@21.1.45/net.neoforged.neoforge.network.PacketDistributor.sendToPlayer(PacketDistributor.java:51) ~[neoforge-21.1.45-universal.jar%23137!/:?] {re:classloading}
	at TRANSFORMER/infernalmobs@1.21.3NF/atomicstryker.infernalmobs.common.InfernalMobsCore.sendVelocityPacket(InfernalMobsCore.java:776) ~[infernalmobs-1.21.3NF.jar%23144!/:1.21.3NF] {re:classloading}
	at TRANSFORMER/infernalmobs@1.21.3NF/atomicstryker.infernalmobs.common.mods.MM_Blastoff.tryAbility(MM_Blastoff.java:66) ~[infernalmobs-1.21.3NF.jar%23144!/:1.21.3NF] {re:classloading}
	at TRANSFORMER/infernalmobs@1.21.3NF/atomicstryker.infernalmobs.common.mods.MM_Blastoff.onUpdate(MM_Blastoff.java:37) ~[infernalmobs-1.21.3NF.jar%23144!/:1.21.3NF] {re:classloading}
	at TRANSFORMER/infernalmobs@1.21.3NF/atomicstryker.infernalmobs.common.MobModifier.onUpdate(MobModifier.java:249) ~[infernalmobs-1.21.3NF.jar%23144!/:1.21.3NF] {re:classloading}
	at TRANSFORMER/infernalmobs@1.21.3NF/atomicstryker.infernalmobs.common.EntityEventHandler.onEntityLivingUpdate(EntityEventHandler.java:175) ~[infernalmobs-1.21.3NF.jar%23144!/:1.21.3NF] {re:classloading}
	at MC-BOOTSTRAP/net.neoforged.bus/net.neoforged.bus.EventBus.post(EventBus.java:350) ~[bus-8.0.1.jar%2367!/:?] {}
	at MC-BOOTSTRAP/net.neoforged.bus/net.neoforged.bus.EventBus.post(EventBus.java:315) ~[bus-8.0.1.jar%2367!/:?] {}
	at TRANSFORMER/neoforge@21.1.45/net.neoforged.neoforge.event.EventHooks.fireEntityTickPost(EventHooks.java:944) ~[neoforge-21.1.45-universal.jar%23137!/:?] {re:classloading}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.level.ServerLevel.tickNonPassenger(ServerLevel.java:775) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.world.level.Level.guardEntityTick(Level.java:570) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.level.ServerLevel.lambda$tick$2(ServerLevel.java:420) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.world.level.entity.EntityTickList.forEach(EntityTickList.java:54) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading}
-- Entity being ticked --
Details:
	Entity Type: minecraft:skeleton (net.minecraft.world.entity.monster.Skeleton)
	Entity ID: 53
	Entity Name: Skeleton
	Entity's Exact location: 971.26, 153.50, 42.05
	Entity's Block location: World: (971,153,42), Section: (at 11,9,10 in 60,9,2; chunk contains blocks 960,-64,32 to 975,319,47), Region: (1,0; contains chunks 32,0 to 63,31, blocks 512,-64,0 to 1023,319,511)
	Entity's Momentum: 0.00, -0.08, 0.00
	Entity's Passengers: []
	Entity's Vehicle: null
Stacktrace:
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.world.level.Level.guardEntityTick(Level.java:570) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.level.ServerLevel.lambda$tick$2(ServerLevel.java:420) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.world.level.entity.EntityTickList.forEach(EntityTickList.java:54) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.level.ServerLevel.tick(ServerLevel.java:400) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1037) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:317) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:917) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:707) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.lambda$spin$2(MinecraftServer.java:267) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at java.base/java.lang.Thread.run(Thread.java:1583) [?:?] {re:mixin}


-- Affected level --
Details:
	All players: 2 total; [ServerPlayer['Naifus'/46, l='ServerLevel[world]', x=981.70, y=153.50, z=41.62], ServerPlayer['saritadinamita'/533, l='ServerLevel[world]', x=980.45, y=153.50, z=42.40]]
	Chunk stats: 2209
	Level dimension: minecraft:overworld
	Level spawn location: World: (1019,104,-75), Section: (at 11,8,5 in 63,6,-5; chunk contains blocks 1008,-64,-80 to 1023,319,-65), Region: (1,-1; contains chunks 32,-32 to 63,-1, blocks 512,-64,-512 to 1023,319,-1)
	Level time: 261019 game time, 36427 day time
	Level name: world
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Commands: false
	Level weather: Rain time: 105651 (now: false), thunder time: 49284 (now: false)
	Known server brands: neoforge
	Removed feature flags: 
	Level was modded: true
	Level storage version: 0x04ABD - Anvil
	Loaded entity count: 390
Stacktrace:
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.level.ServerLevel.fillReportDetails(ServerLevel.java:1718) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1040) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:317) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:917) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:707) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.lambda$spin$2(MinecraftServer.java:267) ~[server-1.21.1-20240808.144430-srg.jar%23136!/:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at java.base/java.lang.Thread.run(Thread.java:1583) [?:?] {re:mixin}


-- System Details --
Details:
	Minecraft Version: 1.21.1
	Minecraft Version ID: 1.21.1
	Operating System: Windows 11 (amd64) version 10.0
	Java Version: 21.0.4, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode, sharing), Oracle Corporation
	Memory: 744826768 bytes (710 MiB) / 1375731712 bytes (1312 MiB) up to 32212254720 bytes (30720 MiB)
	CPUs: 16
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 7 5800H with Radeon Graphics         
	Identifier: AuthenticAMD Family 25 Model 80 Stepping 0
	Microarchitecture: Zen 3
	Frequency (GHz): 3.19
	Number of physical packages: 1
	Number of physical CPUs: 8
	Number of logical CPUs: 16
	Graphics card #0 name: AMD Radeon(TM) Graphics
	Graphics card #0 vendor: Advanced Micro Devices, Inc.
	Graphics card #0 VRAM (MiB): 3072.00
	Graphics card #0 deviceId: VideoController1
	Graphics card #0 versionInfo: 31.0.12027.9001
	Memory slot #0 capacity (MiB): 32768.00
	Memory slot #0 clockSpeed (GHz): 3.20
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MiB): 32768.00
	Memory slot #1 clockSpeed (GHz): 3.20
	Memory slot #1 type: DDR4
	Virtual memory max (MiB): 71602.69
	Virtual memory used (MiB): 8158.68
	Swap memory total (MiB): 9216.00
	Swap memory used (MiB): 4.43
	Space in storage for jna.tmpdir (MiB): <path not set>
	Space in storage for org.lwjgl.system.SharedLibraryExtractPath (MiB): <path not set>
	Space in storage for io.netty.native.workdir (MiB): <path not set>
	Space in storage for java.io.tmpdir (MiB): available: 106174.36, total: 199178.30
	Space in storage for workdir (MiB): available: 1746599.13, total: 1753513.00
	JVM Flags: 1 total; -Xmx30G
	Server Running: true
	Player Count: 2 / 20; [ServerPlayer['Naifus'/46, l='ServerLevel[world]', x=981.70, y=153.50, z=41.62], ServerPlayer['saritadinamita'/533, l='ServerLevel[world]', x=980.45, y=153.50, z=42.40]]
	Active Data Packs: vanilla, mod_data, mod/yungsapi (incompatible), mod/villagesandpillages (incompatible), mod/doubledoors, mod/zombiehorsespawn, mod/worldedit (incompatible), mod/skeletonhorsespawn, mod/terralith, mod/mr_remove_terralithintromessage (incompatible), mod/justmobheads, mod/dungeons_arise_seven_seas (incompatible), mod/reap (incompatible), mod/t_and_t, mod/healingcampfire, mod/collective, mod/infernalmobs (incompatible), mod/neoforge, mod/dungeons_arise, mod/cristellib
	Available Data Packs: bundle, trade_rebalance, vanilla, mod/collective, mod/cristellib, mod/doubledoors, mod/dungeons_arise, mod/dungeons_arise_seven_seas (incompatible), mod/healingcampfire, mod/infernalmobs (incompatible), mod/justmobheads, mod/mr_remove_terralithintromessage (incompatible), mod/neoforge, mod/reap (incompatible), mod/skeletonhorsespawn, mod/t_and_t, mod/terralith, mod/villagesandpillages (incompatible), mod/worldedit (incompatible), mod/yungsapi (incompatible), mod/zombiehorsespawn, mod_data
	Enabled Feature Flags: minecraft:vanilla
	World Generation: Experimental
	World Seed: 5010719563432861189
	Is Modded: Definitely; Server brand changed to 'neoforge'
	Type: Dedicated Server (map_server.txt)
	ModLauncher: 11.0.4+main.d2e20e43
	ModLauncher launch target: forgeserver
	ModLauncher services: 
		sponge-mixin-0.15.2+mixin.0.8.7.jar mixin PLUGINSERVICE 
		loader-4.0.24.jar slf4jfixer PLUGINSERVICE 
		loader-4.0.24.jar runtime_enum_extender PLUGINSERVICE 
		at-modlauncher-10.0.1.jar accesstransformer PLUGINSERVICE 
		loader-4.0.24.jar runtimedistcleaner PLUGINSERVICE 
		modlauncher-11.0.4.jar mixin TRANSFORMATIONSERVICE 
		modlauncher-11.0.4.jar fml TRANSFORMATIONSERVICE 
	FML Language Providers: 
		javafml@4.0
		lowcodefml@4.0
		minecraft@4.0
	Mod List: 
		server-1.21.1-20240808.144430-srg.jar             |Minecraft                     |minecraft                     |1.21.1              |Manifest: NOSIGNATURE
		YungsApi-1.21-NeoForge-5.0.0.jar                  |YUNG's API                    |yungsapi                      |1.21-NeoForge-5.0.0 |Manifest: NOSIGNATURE
		villagesandpillages-neoforge-mc1.21.1-1.0.1.jar   |Villages & Pillages           |villagesandpillages           |1.0.1               |Manifest: NOSIGNATURE
		doubledoors-1.21.1-5.9.jar                        |Double Doors                  |doubledoors                   |5.9                 |Manifest: NOSIGNATURE
		zombiehorsespawn-1.21.1-5.1.jar                   |Zombie Horse Spawn            |zombiehorsespawn              |5.1                 |Manifest: NOSIGNATURE
		worldedit-mod-7.3.6.jar                           |WorldEdit                     |worldedit                     |7.3.6+6892-3d660b8  |Manifest: NOSIGNATURE
		skeletonhorsespawn-1.21.1-4.0.jar                 |Skeleton Horse Spawn          |skeletonhorsespawn            |4.0                 |Manifest: NOSIGNATURE
		Terralith_1.21_v2.5.4.jar                         |Terralith                     |terralith                     |2.5.4               |Manifest: NOSIGNATURE
		remove-terralith-intro-message-1.1.jar            |Remove Terralith Intro Message|mr_remove_terralithintromessag|1.1                 |Manifest: NOSIGNATURE
		justmobheads-1.21.1-8.3.jar                       |Just Mob Heads                |justmobheads                  |8.3                 |Manifest: NOSIGNATURE
		DungeonsAriseSevenSeas-1.21.x-1.0.3.2-neoforge.jar|When Dungeons Arise: Seven Sea|dungeons_arise_seven_seas     |1.0.3.2             |Manifest: NOSIGNATURE
		reap-neoforge-1.21.1-1.1.2.jar                    |Reap Mod                      |reap                          |1.21.1-1.1.2        |Manifest: NOSIGNATURE
		t_and_t-neoforge-fabric-1.13.2.jar                |Towns and Towers              |t_and_t                       |1.13.2              |Manifest: NOSIGNATURE
		healingcampfire-1.21.1-6.1.jar                    |Healing Campfire              |healingcampfire               |6.1                 |Manifest: NOSIGNATURE
		collective-1.21.1-7.84.jar                        |Collective                    |collective                    |7.84                |Manifest: NOSIGNATURE
		infernalmobs-1.21.3NF.jar                         |Infernal Mobs                 |infernalmobs                  |1.21.3NF            |Manifest: NOSIGNATURE
		neoforge-21.1.45-universal.jar                    |NeoForge                      |neoforge                      |21.1.45             |Manifest: NOSIGNATURE
		DungeonsArise-1.21.x-2.1.60-release.jar           |When Dungeons Arise           |dungeons_arise                |2.1.60              |Manifest: NOSIGNATURE
		cristellib-neoforge-1.2.8.jar                     |Cristel Lib                   |cristellib                    |1.2.8               |Manifest: NOSIGNATURE
	Crash Report UUID: dffff9ab-6278-450d-a3c5-599aa35907ab
	FML: 4.0.24
	NeoForge: 21.1.45

```

**Comment by AtomicStryker** [comment]
Infernal Mobs requires clients to have the mod too. I guess this is what your error is saying.

**Comment by AtomicStryker** [comment]
The error was in fact mine which i realized in duplicate tracker https://github.com/AtomicStryker/atomicstrykers-minecraft-mods/issues/523

but it should be fixed in 1.21.1.2NF
